### PR TITLE
Julia 0.7 updates

### PR DESCRIPTION
**Pull request by sidthekid402**
Update deprecated syntax with Julia 0.7 equivalents.

### CODE REVIEW

**Comment by mykelk** [comment]
It looks like this fails on 0.6. I think it is due to the move from `Base.Test` to `Test`. This means that we need to change the `REQUIRE` file (and the .yml file) to specify Julia 0.7. Also, I don't think we want to commit the svg output to the repo.

File: test/runtests.jl [hunk 1]
```diff
@@ -3,19 +3,13 @@ using Test
 
 using TikzPictures
 tp = TikzPicture("\\draw (0,0) -- (10,10);\n\\draw (10,0) -- (0,10);\n\\node at (5,5) {tikz \$\\sqrt{\\pi}\$};", options="scale=0.25", preamble="")
-save(TEX("test"), tp)
+td = TikzDocument([tp],["hello"])
```
Comment (mykelk) [inline test/runtests.jl]: You can get better coverage if you create an empty `TikzDocument` and then add to it using the `push!` command supported by `TikzDocument`.

Comment (sidthekid402) [reply test/runtests.jl]: Will fix now.

File: src/TikzPictures.jl [hunk 2]
```diff
@@ -258,25 +296,34 @@ end
...
-                rm("$filename.pdf")
+                # delete tmp dir
+                rm(tempdir, recursive=true)
             end
         else
-            save(TEX("$filename.tex"), tp)
+            save(TEX("$(filename).tex"), tp)
             if tp.enableWrite18
-                success(`$(tikzCommand()) --enable-write18 --output-format=dvi $filename`)
+                success(`$(tikzCommand()) --enable-write18 --output-format=dvi $(".")`)
```
Comment (mykelk) [inline src/TikzPictures.jl]: Why the `$(".")`?

File: src/TikzPictures.jl [hunk 3]
```diff
@@ -182,23 +182,45 @@ function latexerrormsg(s)
...
 function save(f::PDF, tp::TikzPicture)
-    # Generate the .tex file and make pass along any possible errors
-    foldername = dirname(f.filename)
-    if isempty(foldername)
-        foldername = "."
+    # PDF's are intermediate files for SVG's, so if this function 
+    # is being called inside the save(f::SVG, tp::TikzPicture) function,
+    # the f.filename string would have a temp directory path attached to it
+    # ex - tmp3208yfe/test
+    ind = findfirst(isequal('/'), f.filename)
```
Comment (mykelk) [inline src/TikzPictures.jl]: why is this bit needed now?

File: src/TikzPictures.jl (hunk 2 above)
Comment (sidthekid402) [reply src/TikzPictures.jl]: All the intermediate files are being created in the temp directory, but the tikzCommand() only works when in the same directory as the .tex file. It's messy, but I changed the working directory to the temp directory temporarily and executed the command then switched back.

File: src/TikzPictures.jl (hunk 3 above)
Comment (sidthekid402) [reply src/TikzPictures.jl]: Was doing something earlier with it. It isn't needed. I'll remove it with this next commit.

File: src/TikzPictures.jl (hunk 2 above)
Comment (mykelk) [reply src/TikzPictures.jl]: Why not just make `$(".")` just `.`?

Comment (sidthekid402) [reply src/TikzPictures.jl]: I was just replacing the text inside the parens. I'll change it now.

**Pull request merged**